projects/js-packages/components/components/pricing-table/styles.module.scss

```diff
@@ -0,0 +1,102 @@
+.container {
+	--padding: 32px;
+	color: var( --jp-black );
+}
+
+.table {
+	--gap: 24px;
+	position: relative;
+	z-index: 2;
+	padding: var( --padding ) 0;
+
+	.is-viewport-large & {
+		display: grid;
+		grid-template-columns: repeat( var( --columns ), 1fr );
+		grid-auto-flow: column;
+		grid-template-rows: repeat( var( --rows ), minmax( min-content, max-content ) );
+		column-gap: var( --gap );
+	}
+}
+
+.card {
+	background: var( --jp-white );
+	box-shadow: 0px 4px 24px rgba( 0, 0, 0, 0.05 ), inset 0px 0px 0px 1px var( --jp-gray );
+	border-radius: 4px;
+	padding: var( --padding ) 0;
+	margin-top: var( --padding );
+
+	& + & {
+		margin-top: var( --gap );
+	}
+}
+
+.header {
+	margin-bottom: var( --padding );
+	padding: 0 var( --padding );
+
+	.is-viewport-large & {
+		&::before {
+			top: 0;
+			position: absolute;
+			height: 100%;
+			width: calc(
+				( 100% / var( --columns ) ) - ( ( var( --columns ) - 1 ) * var( --gap ) / var( --columns ) )
+			);
+			content: '';
+			background: var( --jp-white );
+			box-shadow: 0px 4px 24px rgba( 0, 0, 0, 0.05 ), inset 0px 0px 0px 1px var( --jp-gray );
+			border-radius: 4px;
+			z-index: -1;
+			transform: translateX( calc( var( --padding ) * -1 ) );
+		}
+	}
+}
+
+.item {
+	display: flex;
+	align-items: center;
+
+	&::before {
+		content: '';
+		width: 24px;
+		height: 24px;
+		display: block;
+		background-image: url( 'data:image/svg+xml;utf8,%3Csvg%20fill%3D%27none%27%20xmlns%3D%27http%3A//www.w3.org/2000/svg%27%20viewBox%3D%270%200%2024%2024%27%3E%3Ccircle%20cx%3D%2712%27%20cy%3D%2712%27%20r%3D%278%27%20stroke%3D%27%23A7AAAD%27%20stroke-width%3D%271.5%27/%3E%3Cpath%20fill%3D%27%23A7AAAD%27%20d%3D%27M11%2011h2v6h-2zM11%207h2v2h-2z%27/%3E%3C/svg%3E' );
+		background-size: cover;
+		background-position: center;
+		margin-right: var( --spacing-base );
+		flex-shrink: 0;
+	}
+
+	& + & {
+		border-top: 1px solid var( --jp-gray );
+		margin-top: 16px;
+		padding-top: 16px;
+	}
+}
+
+.label.label {
+	font-weight: 700;
```

Comment:
Yeah, I think I added the `strong` later (that's also why I initially used `.value.value` and `.label.label`, since without those double classes the `Text` component was overriding it.